taskfile/read/taskfile.go

```diff
@@ -69,19 +80,23 @@ func Taskfile(readerNode *ReaderNode) (*taskfile.Taskfile, error) {
 				Optional:       includedTask.Optional,
 				AdvancedImport: includedTask.AdvancedImport,
 				Vars:           includedTask.Vars,
+				BaseDir:        includedTask.BaseDir,
 			}
 			if err := tr.Err(); err != nil {
 				return err
 			}
 		}
 
-		path, err := execext.Expand(includedTask.Taskfile)
+		tf, err := includedTask.FullTaskfilepath()
```

Comment:
This diff makes it look as though `execext.Expand()` has been removed - it's still present, just lower down on line #95.